src/components/WorkOrder/Photos/hooks/uploadFiles/faultTolerantRequest.ts

```diff
@@ -0,0 +1,36 @@
+import retry from 'retry'
+
+const DEFAULT_NUMBER_OF_RETRIES = 3
+
+const faultTolerantRequest = async (
```

Comment:
I'd be confused if I saw this without context in future

```suggestion
/**
 * Returns a promise that tries to successfully resolve
 * a specified amount of times with exponential backoff
 */
const faultTolerantRequest = async (
```